src/main/webapp/database-tables.js

```diff
@@ -1,82 +1,46 @@
+let tablesList = [];
+
 //Boolean to ensure data only shows after first click
-var showing = Boolean(false)
+var showing = Boolean(false);
 
 function showDatabase() {
-  if (!showing) { 
+  if (!showing) {
     showing = true;
     const search = window.location.search;
     const queryString = '/data-from-db' + search;
+    document.getElementById("tables").innerText = 'Loading...';
 
+    tablesList = [];
+  
     fetch(queryString)
     .then(response => response.json())
     .then((data) => { 
-      const dataArea = document.getElementById("data");
-    
+      document.getElementById("tables").innerText = '';
+      document.getElementById("sql").innerText = '';
+      
+      let id = 0;
```

Comment:
What does this id variable do?

Reply:
It gives each table an id, makes it easier with sorting + rendering